assets/src/components/mapMarkers.tsx

```diff
@@ -257,72 +257,108 @@ export const StationMarker = React.memo(
   }
 )
 
-export const RouteStopMarkers = ({
+const uniqueStopsByLocation = (stops: Stop[]): Stop[] => {
+  let uniqueLocationToStop: Record<string, Stop> = {}
+  // When there are two stops with the same location, prefer the latest station
```

Comment:
This is to avoid situations where a station & child stop are co-located, but the child stop was rendered on top at close zoom levels.